futures-util/src/io/empty.rs

```diff
@@ -0,0 +1,64 @@
+use futures_core::task::{Context, Poll};
+use futures_io::{AsyncBufRead, AsyncRead, Initializer};
+use std::fmt;
+use std::io;
+use std::pin::Pin;
+
+/// Stream for the [`empty()`] function.
+#[must_use = "streams do nothing unless polled"]
```

Comment:
`s/stream/reader` for these two lines, and similar across the other files.

Reply:
Fixed.